testpilot/frontend/static-src/app/views/feedback-view.js

```diff
@@ -1,84 +1,45 @@
-import app from 'ampersand-app';
 import BaseView from './base-view';
```

Comment:
Do we still need this feedback subview? I think the feedback button is just supposed to open the SurveyMonkey URL in a new tab without a dialog modal.